src/components/footnote-group/footnote-group.scss

```
@@ -0,0 +1,28 @@
+// TODO: Remove unused
+@use '../../compiled/tokens/scss/color';
+@use '../../compiled/tokens/scss/size';
+@use '../../mixins/focus';
+@use '../../mixins/ms';
+
+/// Footnote groups are used to group footnotes together in a list
+.c-footnote-group {
+  --spacing-footnotes: #{ms.step(2)};
+
+  border-top: size.$edge-medium solid color.$base-gray-lighter;
+  margin-top: var(--spacing-footnotes);
+  padding-top: var(--spacing-footnotes);
+}
+
+.c-footnote-group__list > * + * {
+  margin-top: var(--spacing-footnotes);
+}
+
+/// Compact footnote groups have smaller text, smaller gaps, and no top
+/// border/padding
+.c-footnote-group__compact {
```

@Paul-Hebert Should this be a modifier? 🤔 
```suggestion
.c-footnote-group--compact {
```